compositor_common/src/scene/builtin_transformations.rs

```diff
@@ -22,6 +24,13 @@ pub enum BuiltinSpec {
         #[serde(default)]
         background_color_rgba: RGBAColor,
     },
+    TiledLayout {
+        #[serde(default)]
+        background_color_rgba: RGBAColor,
```

Comment:
Is default fully transparent?

Reply:
Yep

Reply:
I planned to add an option to set the background as a node, allowing to set anything that node can produce as a background, but I'm not sure whether mixing it in the next transformation (sth like blend), wouldn't be better. What do you think?

Reply:
Yes, this should be the responsibility of a different component, either using fixed position layout or we could make an Overlay component with a dynamic resolution that takes 2 inputs  where (resolution is based on one of them)